metrics/prometheus_exporter.go

```diff
@@ -82,7 +82,7 @@ func startNewPromSrv(e *prom.Exporter, port int) *http.Server {
 		curPromSrv.Close()
 	}
 	curPromSrv = &http.Server{
-		Addr:    fmt.Sprint(":", port),
+		Addr:    fmt.Sprintf("%s:%d", host, port),
```

Comment:
`host + ":" + strconv.Itoa(port)`
But up to you, this is going to be called once probably in the lifetime of the service.